### PR TITLE
MM-44323: disable duplicate if disallowed

### PR DESCRIPTION
**Pull request by lieut-data**
#### Summary
If a team disallows public playbook creation, disable the duplicate option for playbooks on that team:

CleanShot 2022-05-17 at 16 38 55@2x

I investigated adding a unit test, but found the effort to customize the system scheme or team scheme out of scope for this small fix.

#### Ticket Link
Fixes: https://mattermost.atlassian.net/browse/MM-44323

#### Checklist
- [ ] ~~Telemetry updated~~
- [ ] ~~Gated by experimental feature flag~~
- [ ] ~~Unit tests updated~~

### CODE REVIEW

File: webapp/src/components/dot_menu.tsx [hunk 1]
```diff
@@ -164,15 +164,43 @@ export const DropdownMenuItemStyled = styled.a`
...
+    cursor: default;
+    font-family: 'Open Sans';
+    font-style: normal;
+    font-weight: normal;
+    font-size: 14px;
+    color: var(--center-channel-color-40);
+    padding: 10px 20px;
+    text-decoration: unset;
+}
+`;
```
Comment (agarciamontoro) [inline webapp/src/components/dot_menu.tsx:190]: I'm not sure I understand this, aren't the styles duplicated between `DisabledDropdownMenuItemStyled` and `DropdownMenuItemStyled`? We could simply pass the `props.disabled` to `DropdownMenuItemStyled` and add the different `color` and `cursor` if that prop is true.
We can also just avoid the different return in `DropdownMenuItem` below if we pass `props.disabledAltText` conditionally to `title`, right?

**Review by agarciamontoro — CHANGES_REQUESTED** [review]
The logic is good, but I have a doubt about the CSS below.

File: webapp/src/components/dot_menu.tsx (hunk 1 above)
Comment (agarciamontoro) [reply webapp/src/components/dot_menu.tsx:190]: Ah, I didn't catch that. It makes sense to have two returns, then. Thanks for the context :)

**Comment by lieut-data** [comment]
Thanks again for the help, @calebroseland! Now looks like:

image

**Review by calebroseland — APPROVED** [review]
Looks good! For expediency, I pushed a brief set of changes:
- Intl
- Added to playbook editor
- (while there) Added a small UX tweak from feedback: default to outline tab as duplicated playbooks will have zero usage on first load

CleanShot 2022-05-31 at 17 01 30@2x

File: webapp/i18n/en.json [hunk 2]
```diff
@@ -250,6 +250,7 @@
   "Sx3lHL": "Integer",
   "T5rX+W": "How often should an update be posted?",
   "TBez4r": "There are no playbooks to view. You don't have permission to create playbooks in this workspace.",
+  "TD8WrM": "Duplicate is disabled for this team.",
```
Comment (lieut-data) [inline webapp/i18n/en.json:253]: Sigh, I had done this correctly in an earlier cherry-pick, but forgot after cleaning up. Thank you!

**Review by agarciamontoro — APPROVED** [review]
:tada:

**Pull request merged**